addons/account/models/account_journal.py

```diff
@@ -104,7 +104,8 @@ def _default_invoice_reference_model(self):
     invoice_reference_model = fields.Selection(string='Communication Standard', required=True, selection=[('odoo', 'Odoo'), ('euro', 'European')], default=_default_invoice_reference_model, help="You can choose different models for each type of reference. The default one is the Odoo reference.")
 
     #groups_id = fields.Many2many('res.groups', 'account_journal_group_rel', 'journal_id', 'group_id', string='Groups')
-    currency_id = fields.Many2one('res.currency', help='The currency used to enter statement', string="Currency")
+    currency_id = fields.Many2one('res.currency', help="- Currency used to enter Bank or Cash statements.\n"
+                                  "- Default currency of each new move on all other journal types", string="Currency", compute='_compute_journal_currency', store=True, readonly=False, required=True, precompute=True)
```

Comment:
this indentation is chaotic
```py
currency_id = fields.Many2one(
    comodel_name='res.currency',
    string="Currency",
    compute='_compute_journal_currency',
    store=True,
    readonly=False,
    required=True,
    precompute=True,
    help="The default currency suggested on journal entries or the currency for bank transactions",
)
```
(changed the helper, what you do think?)

Reply:
I think this helper is ok, I copied what @tsb-odoo wrote to me :slightly_smiling_face: